deploy.sh

```diff
@@ -6,7 +6,7 @@ base_dir="$(dirname $(realpath $0))"
 cd "$base_dir"
 
 flutter clean
-flutter build web --release --web-renderer canvaskit
+flutter build web --release
```

Comment:
I've removed the web renderer setting to use auto choice.
https://docs.flutter.dev/deployment/web#choosing-a-web-renderer